ops/terraform/services/server/server-regression/iam.tf

```
@@ -326,3 +326,86 @@ resource "aws_iam_role" "glue_trigger" {
     aws_iam_policy.glue.arn
   ]
 }
+
+resource "aws_iam_role" "spice_refresh_role" {
+  name = "bfd-${local.env}-${local.service}-spice-trigger-role"
+  assume_role_policy = jsonencode(
+    {
+      Statement = [
+        {
+          Action = "sts:AssumeRole"
+          Effect = "Allow"
+          Principal = {
+            Service = "lambda.amazonaws.com"
+          }
+        },
+      ]
+      Version = "2012-10-17"
+    }
+  )
+  force_detach_policies = true
+}
+
+resource "aws_iam_policy" "spice_refresh_logs_policy" {
+  name = "bfd-${local.env}-${local.service}-refresh-spice-trigger-logs"
+  description = join("", [
+    "Permissions for the bfd-${local.env}-${local.service}-refresh-spice-trigger Lambda to write to its corresponding ",
+    "CloudWatch Log Group and Log Stream",
+  ])
+  policy = jsonencode(
+    {
+      Statement = [
+        {
+          Action   = "logs:CreateLogGroup"
+          Effect   = "Allow"
+          Resource = "arn:aws:logs:${local.region}:${local.account_id}:*"
+        },
+        {
+          Action = [
+            "logs:CreateLogStream",
+            "logs:PutLogEvents",
+          ]
+          Effect = "Allow"
+          Resource = [
+            "arn:aws:logs:${local.region}:${local.account_id}:log-group:/aws/lambda/${local.spice_trigger_lambda_name}:*",
+          ]
+        },
+      ]
+      Version = "2012-10-17"
+    }
+  )
+}
+
+resource "aws_iam_policy" "spice_refresh_policy" {
+  name = "bfd-${local.env}-${local.service}-quicksight-spice-permissions"
```

```suggestion
  name = "${local.spice_trigger_lambda_name}-quicksight"
```

This is less of a nit than the previous ones, as it is not really possible to infer the target of the IAM Policy from its name. Changing it to this would indicate that the Policy is attached to the SPICE Lambda and that it pertains to QuickSight.